test/workbox-precaching/integration/precache-and-update.js

```diff
@@ -0,0 +1,13 @@
+describe(`[workbox-precaching] Precache and Update`, function() {
```

Comment:
Do you want to put something about this being an Integration test in the `describe()` comment?

Reply:
It should be somewhat clear it's an integration test because of the gulp logs surrounding the mocha results.

If you think it'll be helpful happy to add a notion of `[workbox-precaching] {<Test Type>} <Test Title>` to our top level describe calls.